File: SSZS/src/ws/hoyland/sszs/PauseObject.java
package ws.hoyland.sszs;

public class PauseObject {
	
	private static PauseObject instance;
	
	private PauseObject(){
		
	}
	
	public static PauseObject getInstance(){
		if(instance==null){
			instance = new PauseObject();
		}
		return instance;
	}
}

File: trunk/sszs-sln/sszs/PauseObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ws.hoyland.sszs
{
    public class PauseObject
    {
        private static PauseObject instance;

        private PauseObject()
        {

        }

        public static PauseObject getInstance()
        {
            if (instance == null)
            {
                instance = new PauseObject();
            }
            return instance;
        }
    }
}
